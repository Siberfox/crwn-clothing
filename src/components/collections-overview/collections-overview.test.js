import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import { CollectionsOverview } from "./collections-overview.component";

it("should render CollectionsOverview component", () => {
  const mockCollections = [{ id: "idkfa" }, { id: "iddqd" }];
  const wrapper = shallow(
    <CollectionsOverview collections={mockCollections} />
  );
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
