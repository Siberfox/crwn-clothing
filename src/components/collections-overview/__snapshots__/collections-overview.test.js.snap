// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CollectionsOverview component 1`] = `
<styled.div>
  <withRouter(CollectionPreview)
    key="idkfa"
  />
  <withRouter(CollectionPreview)
    key="iddqd"
  />
</styled.div>
`;
